draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1721,13 +1728,16 @@ cookies are disabled, the user agent MUST NOT include a Cookie header in
 outbound HTTP requests and the user agent MUST NOT process Set-Cookie headers
 in inbound HTTP responses.
 
-Some user agents provide users the option of preventing persistent storage of
+User agents MAY offer a way to change the cookie policy (see
+{{ignoring-cookies}}).
+
+User agents MAY provide users the option of preventing persistent storage of
 cookies across sessions. When configured thusly, user agents MUST treat all
 received cookies as if the persistent-flag were set to false. Some popular
 user agents expose this functionality via "private browsing" mode
 {{Aggarwal2010}}.
 
-Some user agents provide users with the ability to approve individual writes to
+User agents MAY provide users with the ability to approve individual writes to
```

Comment:
I wonder if it's reasonable to take this out. I don't think we actually want to encourage any browser to ask users to individually approve writes to the cookie jar.

Reply:
I agree. Removed.